NeuroRuler/utils/parser.py

```diff
@@ -13,6 +13,19 @@
 """Dict of settings resulting from JSON file parsing. Global within this file."""
 
 
+def parse_cli_cli() -> None: # TODO: better name? lol -Eric
```

Comment:
This can be renamed to `parse_cli`.

As mentioned in an above comment, we can shrimply remove `parse_gui_cli` because the user can input GUI settings via the JSON file.

Reply:
Might leave that for a different PR